themes/default/content/what-is/what-is-infrastructure-as-code.md

```diff
@@ -62,85 +62,105 @@ video:
 
-The term “infrastructure as code” or IaC has become commonplace, but there are important questions attached to it. What is infrastructure as code really? What are the benefits of infrastructure as code?  How do you integrate infrastructure as code into your organization? In this article, we’re going to touch on all these points and discuss why infrastructure as code matters for modern application development.
+In this article, we'll touch on three key aspects regarding infrastructure as code:
 
-## Why Has Infrastructure as Code Become Important?
+* Why is infrastructure as code important?
+* What are the key elements of infrastructure as code?
+* What benefits does infrastructure as code provide?
+* How do you get started with infrastructure as code?
 
-Infrastructure as code matters because of three significant trends, all of them happening at the same time.
+Let's start with examing why infrastructure as code is important.
 
-### The transition to the cloud
+## Why is Infrastructure as Code Important?
+
+Infrastructure as code is important because of three significant trends, all of them happening at the same time.
+
+### The Transition to the Cloud
 
 One trend, of course, is the ongoing transition to the cloud. More and more companies are shifting workloads from on-premises infrastructure to cloud environments. Cloud-based infrastructure is provisioned via APIs, and, as a result, can be easily managed with infrastructure as code tools.
 
-### Cloud modernization
+### Cloud Modernization
+
+The second trend is cloud modernization. After organizations migrate to the cloud, they tend to look for opportunities to maximize the value they get from their cloud environment. This frequently involves adopting technologies such as serverless, containers and Kubernetes. When applied correctly, these technologies enable teams to deliver value more quickly. In some cases, cloud modernization involves using managed services to offload some of the heavy lifting to the cloud provider. These technologies and services generally require a more granular management of infrastructure. Stitching together all the primitives that the cloud provider offers into solutions that serve the business is a great fit for infrastructure as code.
```

Comment:
should we call out the ephemerality and statelessness of some of these compute primitives?

Reply:
Yes, that's probably a good idea (and my response to your comment above also gives me some ideas for additional changes in this section).